tests/e2e/tests/test_sanity.py

```diff
@@ -224,3 +244,30 @@ def test_katib_experiment(self, setup, cluster, region):
             raise AssertionError("Expected K8sApiException Not Found")
         except K8sApiException as e:
             assert "Not Found" == e.reason
+
+    def test_kserve_with_irsa(self, region, metadata, kfp_client, clone_tensorflow_serving, kserve_iam_service_account, kserve_secret, s3_bucket_with_training_data, kserve_inference_service):
+        #Edit the ConfigMap to change the default domain as per your deployment
+        #kubectl edit configmap config-domain -n knative-serving
+        exec_shell(f"kubectl get configmap config-domain -n knative-serving -o yaml > {KNATIVE_CONFIG_FILE}")
+        subdomain = metadata.get("lb_deps").get("config").get("route53").get("subDomain").get("name")
+        knative_config = load_yaml_file(KNATIVE_CONFIG_FILE)
+        knative_config["data"][subdomain] = ""
+        del knative_config["data"]["_example"]
+        write_yaml_file(knative_config, KNATIVE_CONFIG_FILE)
+        kubectl_apply(KNATIVE_CONFIG_FILE)
+
+
+        #export env with subprocess
+        #run inference_sample.py
+
+        os.environ['KUBEFLOW_DOMAIN'] = subdomain
+        os.environ['PROFILE_NAMESPACE'] = "kubeflow-example-com"
+        os.environ['MODEL_NAME'] = "half-plus-two"
+        os.environ['AUTH_PROVIDER'] = "dex"
+
+        
+        assert run_inference_sample() == 200
```

Comment:
this is ok but not very robust, can we use similar method like in setup_load_balancer fixture to run the script using subprocess instead of importing it?

```
cmd = "python utils/load_balancer/setup_load_balancer.py".split()
        retcode = subprocess.call(
            cmd, stderr=subprocess.STDOUT, env=env_value
        )
        assert retcode == 0
```

we can change the kserve script to do
```
    if status_code == 200:
        print("JSON Response ", json.dumps(response.json(), indent=2))
   else:
       raise Exception("prediction failed, status code = ")
```